utils/lists.py

```diff
@@ -10,21 +10,13 @@
 _all_questions = [
     QuestionAnswer("What is the capital city of Spain?", "Madrid"),
     QuestionAnswer("What is arachnophobia the fear of?", "Spiders"),
-    QuestionAnswer("Stockholm is the capital and largest city of what country?", "Sweden"),
-    QuestionAnswer("What year did the website Facebook launch?", "2004"),
-    QuestionAnswer("What is the scientific name for the Southern Lights?", "Aurora Australis"),
-    QuestionAnswer("On average, Americans consume 100 pounds of what per second?", "Chocolate"),
-    QuestionAnswer("How many bones are there in the human body?", "206"),
-    QuestionAnswer("What does Na stand for on the periodic table?", "Sodium"),
-    QuestionAnswer("Which fictional city is the home of Batman?", "Gotham"),
-    QuestionAnswer("Which planet is closest to Earth?", "Venus"),
-    QuestionAnswer("In the US version of The Office, what is the name of the city they live in?", "Scranton"),
-    QuestionAnswer("Which U.S. State is the largest (in terms of land area)?", "Alaska"),
-    QuestionAnswer("Which U.S. State is known as the Sunflower State?", "Kansas"),
-    QuestionAnswer("Who founded Microsoft?", "Bill Gates"),
-    QuestionAnswer("What plant is known to help heal a sunburn?", "Aloe"),
-    QuestionAnswer("How many planets are in our solar system? (Including Pluto!)", "9"),    
-
+    QuestionAnswer("What is your body’s largest organ?", "Skin"),
+    QuestionAnswer("What year did the Titanic movie come out?", "1997"),
+    QuestionAnswer("Who played Neo in The Matrix?", "Keanu Reeves"),
+    QuestionAnswer("What year was the first IPhone released?", "2007"),
+    QuestionAnswer("Which bone are babies born without?", "Knee Cap"),
+    QuestionAnswer("What is Hawkeye's real Name?", "Clint Barton"),
+    QuestionAnswer("Mexico’s Dia de los Muertos, means what in English?", "Day of the Dead"),
```

Comment:
You're working off an old version of `main`. Please pull the latest commits to `main` and rebase your work onto the top of the current `main` branch. You will have to fix a conflict in the process, then force-push.

To start:
```sh
git checkout main
git pull
git checkout max
git rebase main
```

You will then have a merge conflict to fix. This guide might be helpful: http://gitforteams.com/resources/rebasing.html Make sure to keep all of the trivia questions from both changes (order doesn't matter), remove the conflict markers, then `git add utils/lists.py` and `git rebase --continue`.

Then your local branch and the remote branch will have diverged, so run `git push --force-with-lease` to do a forced update to this pull request.

Reply:
This is looking good, but may want to hold off on merging this for a little bit. The format of how we will be reading questions in will now be from a text file. So when that gets merged, would probably be better to put these questions in the trivia_questions_answer.txt file.

Reply:
Fixes look good, thanks! I'll leave the conflict fix up to Seth as I think it will be simpler that way.